Add community build blobfree image for Fairphone 2 (#1)

novski opened the pull request:
Add community build blobfree image for Fairphone 2 (#1)
Bug ID: 1285793
Tested on 1.3.6, 1.4.2, 1.5.1

novski committed:
Add community build blobfree image for Fairphone 2 (#1)

Add community build blobfree image for Fairphone 2 (#1)
Bug ID: 1285793
Tested on 1.3.6, 1.4.2, 1.5.1

lissyx commented on builds.json:
```diff
@@ -148,10 +148,15 @@
         "description": "B2G Nightly user build for Fairphone 2",
         "url": "https://github.com/atilag/b2g-installer-builds/releases/download/b2g-fp2-nightly/FP2.blobfree-dist.zip"
       }
+     {
```
It misses ","

novski committed:
Add community build blobfree image for Fairphone 2 (#1)

Now with the coma. And version 1.2.8 also tested.
1.2.8, 1.3.6, 1.4.2, 1.5.1 are tested.

lissyx commented on builds.json:
```diff
@@ -147,11 +147,16 @@
         "description": "B2G Nightly user build for Fairphone 2",
         "url": "https://github.com/atilag/b2g-installer-builds/releases/download/b2g-fp2-nightly/FP2.blobfree-dist.zip"
+      },
+     {
```
Please fix identation

lissyx commented on builds.json:
```diff
@@ -147,11 +147,16 @@
     "adb": {
       "ro.product.device": [ "FP2" ],
-      "ro.build.display.id": [ "r4275.1_FP2_gms43_1.2.8" ]
+      "ro.build.display.id": [ "r4275.1_FP2_gms43_1.2.8", "r4275.1_FP2_gms54_1.3.6", "r4275.1_FP2_gms56_1.4.2", "r4275.1_FP2_gms59_1.5.1" ]
```
I really think we should just have "r4275.1_FP2_gms59_1.5.1" and ask people to perform stock upgrades before: they can contain important low-level fixes that we might need.

novski committed:
Add community build blobfree image for Fairphone 2 (#1)

Add community build blobfree image for Fairphone 2 (#1)
Bug ID: 1285793
Tested on: 1.2.8, 1.3.6, 1.4.2, 1.5.1
add of correct URL: https://github.com/mozilla-b2g/b2g-installer-builds/releases/download/b2gos-experimental-fp2/FP2.blobfree-dist.zip

novski committed:
Add community build blobfree image for Fairphone 2 (#1)

Add community build blobfree image for Fairphone 2 (#1)
Bug ID: 1285793
Tested on:1.1.7, 1.2.8, 1.3.6, 1.4.2, 1.5.1
add of delivery OS Build-Version 1.1.7
